### ISSUE TITLE
Viewer group usernames case sensitive?

### ISSUE THREAD

**Issue by awrebels** [Firebottle/Firebot#352]
#### Describe your issue or feature request (details are helpful!):

When setting up a custom viewer group I've noticed that the usernames must be case sensitive to work. From what I know mixer names are case insensitive, correct?

-----------------------------------------------------------------------
#### If you're reporting an issue, *please* fill out & do the following:

- What version of Firebot are you running?
4.1.0

- What are the steps to reproduce the issue?
1) create custom viewer group 
2) add a username without case (use a username that displays varying cases in mixer)
3) Test viewer group

- (IMPORTANT) Go to Firebot's root folder (**Settings tab -> Open Root Folder**) then zip the following two files and attach that zip to this issue:

1) In **user-settings > controls**, find the .json file with the name of the board you are having issues with.
2) In **user-settings > logs**, find the latest log file.

```
NOTE: A lot of issues can be solved by,
- Updating to the latest version
- Trying the troubleshooting steps on the wiki
```

**Comment by Firebottle** [comment]
Yes, we should be able to make it so the case does not matter.

**Comment by ebiggz** [comment]
@awrebels Fixed in 4.2.2.
